### PR TITLE
Let them eat filter

### PR DESCRIPTION
**Pull request by broskoski**
Gives the channel / profile filter to everyone, regardless of economic status

### CODE REVIEW

File: apps/user/templates/channels.jade [hunk 1]
```diff
@@ -3,11 +3,10 @@ extends ../../../components/layout/index
 block body
   include _path
   .UserChannels.js-user-channels
-    if user && user.isPremium()
```
Comment (dzucconi) [inline apps/user/templates/channels.jade]: muchsecurity

File: components/filter/index.styl [hunk 2]
```diff
@@ -13,7 +13,7 @@
     position absolute
     height 100%
     width 50px
-    left 10px
+    left 0px
```
Comment (dzucconi) [inline components/filter/index.styl]: 0 miles, 0 pounds, 0 dollars.

What do they have in common?

File: apps/user/templates/channels.jade [hunk 3]
```diff
@@ -3,11 +3,10 @@ extends ../../../components/layout/index
...
-      .flex-container
-        - searchee = (user && author.id == user.id) ? "your " : author.get('username') + "'s public "
-        - title = searchee + "channels"
-        include ../../../components/filter/templates/search_bar
+    .flex-container
+      - searchee = (user && author.id == user.id) ? "your " : author.get('username') + "'s public "
+      - title = searchee + "channels"
```
Comment (dzucconi) [inline apps/user/templates/channels.jade]: searcheeeeeee

Comment (dzucconi) [reply apps/user/templates/channels.jade]: searché

File: components/filter/index.styl (hunk 2 above)
Comment (broskoski) [reply components/filter/index.styl]: Gas, grass or ass, no one rides for free

File: apps/user/templates/channels.jade (hunk 3 above)
Comment (broskoski) [reply apps/user/templates/channels.jade]: One who is being searched on is a searchee

Comment (dzucconi) [reply apps/user/templates/channels.jade]: s̩̀̏̒̇e̘͇͎ͯ̂̋ͅă̜̣͆̐ͥr̴͔̝͖͎̂͆c̆̿ͩ̚h̫͓̦̾͒̍͌̃e̝͖e̙̔ͦ͗̋̍͆̐

**Pull request merged**